apps/docs/content/guides/auth/row-level-security.mdx

```diff
@@ -125,6 +125,20 @@ on my_table for select
 using ( true );
 ```
 
+### Protect views using `security_invoker`
+
+Views do _not_ obey RLS policies by default. This is a feature of Postgres, which automatically creates views with `security definer`.
```

Comment:
It is not 100% accurate to say that `security invoker` views don't follow RLS.

`security definer` means that the view follows the permissions of the role that created the view, rather than the role that is querying the view.

In almost all cases, the view is created by the `postgres` user, which bypasses RLS. For that reason, these views to not respect RLS rules, BUT if a role without RLS created the view, it would still follow those rules.

Reply:
Very good point, thanks for the clarification!